src/styles/components/_buttons.scss

```diff
@@ -0,0 +1,69 @@
+@import "../styles";
+
+@mixin button-block  {
+  display: flex;
+}
+
+@mixin button-text {
+  font-family: $sans-serif-stack;
+  letter-spacing: 0.1em;
+  text-transform: uppercase;
+}
+
+.btn-block {
+  @include button-block;
+}
+
+.btn--sm {
+  @include button-text;
+  border-radius: 3px;
+  font-size: 10px;
+  font-weight: 700;
+  line-height: 16px;
+  padding: 6px 8px 8px 10px;
+  transition: $transition-default;
+}
+
+.btn--md  {
+  @include button-text;
+  border-radius: 3px;
+  font-size: 12px;
+  padding: 13px 14px 15px 14px;
+}
+
+.btn--lg {
+  font-weight: 700;
```

Comment:
not sure which button this is, but just double checking that it doesn't contain text or a border radius.